python/unitytrainers/ppo/trainer.py

```diff
@@ -360,8 +359,8 @@ def update_model(self):
                              self.model.all_old_probs: np.array(
                                  _buffer['action_probs'][start:end]).reshape([-1, self.brain.vector_action_space_size])}
                 if self.is_continuous_action:
-                    feed_dict[self.model.epsilon] = np.array(
-                        _buffer['epsilons'][start:end]).reshape([-1, self.brain.vector_action_space_size])
+                    feed_dict[self.model.output] = np.array(
+                        _buffer['actions'][start:end]).reshape([-1, self.model.a_size])
```

Comment:
You are setting the output of the model ? This does not seem right, can you explain what you did ?

Reply:
You set the output, and the backpropagation process calculates what was required to achieve it. This avoids having to cache the random values used in the sampling.

Reply:
@awjuliani Your response above would be a great code comment :)